Frontend/my-app/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
Let's replace the README with some relevant content. For now, something like
```md
# Project Name
description...
```
should be fine.